monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc

```diff
@@ -188,6 +190,8 @@ Appears in: link:#clustermonitoringconfiguration[ClusterMonitoringConfiguration]
 
 |tolerations|[]v1.Toleration|Defines tolerations for the pods.
 
+|topologySpreadConstraints|[]v1.TopologySpreadConstraint|Defines a pod's topology spread constraints.
```

Comment:
missed `resources` setting for K8sPrometheusAdapter
https://github.com/openshift/cluster-monitoring-operator/blob/release-4.14/pkg/manifests/types.go#L113